eo-maven-plugin/src/test/java/org/eolang/maven/MjLintTest.java

```diff
@@ -26,11 +26,6 @@
  * Test cases for {@link MjLint}.
  *
  * @since 0.31.0
- * @todo #4049:30min Replace all occurrences of new XMLDocument().nodes() with new Xnav().path().
```

Comment:
@vl4ds4m maybe I made a mistake in todo. Such occurrences must be replaced not only in `MjLintTest`, but in the all project. So could you please update todo or create a new one?